### ISSUE TITLE
Lots of crashes for 2.2.3

### ISSUE THREAD

**Issue by natsumi** [koekeishiya/yabai#412]
Ever since I updated to the latest release, I've been experiencing a lot of crashes.  

I've uninstalled and reinstalled both yabai and sa

```
xception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000008
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Segmentation fault: 11
Termination Reason:    Namespace SIGNAL, Code 0xb
Terminating Process:   exc handler [26286]

VM Regions Near 0x8:
--> 
    __TEXT                 0000000105406000-0000000105439000 [  204K] r-x/r-x SM=COW  /usr/local/Cellar/yabai/2.2.3/bin/yabai

Thread 0:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	0x00007fff6754125a mach_msg_trap + 10
1   libsystem_kernel.dylib        	0x00007fff675415d0 mach_msg + 60
2   com.apple.CoreFoundation      	0x00007fff2fda5d0b __CFRunLoopServiceMachPort + 322
3   com.apple.CoreFoundation      	0x00007fff2fda48e7 __CFRunLoopRun + 1695
4   com.apple.CoreFoundation      	0x00007fff2fda3bd3 CFRunLoopRunSpecific + 499
5   com.apple.CoreFoundation      	0x00007fff2fe2c19a CFRunLoopRun + 40
6   yabai                         	0x000000010542c829 main + 1993
7   libdyld.dylib                 	0x00007fff674007fd start + 1

Thread 1 Crashed:
0   yabai                         	0x000000010541a6d2 window_title + 18
1   yabai                         	0x0000000105409959 event_signal_transmit + 1641
2   yabai                         	0x000000010540923d event_loop_run + 109
3   libsystem_pthread.dylib       	0x00007fff67604e65 _pthread_start + 148
4   libsystem_pthread.dylib       	0x00007fff6760083b thread_start + 15

Thread 2:
0   libsystem_kernel.dylib        	0x00007fff6754787e __accept + 10
1   yabai                         	0x0000000105407c1c socket_connection_handler + 44
2   libsystem_pthread.dylib       	0x00007fff67604e65 _pthread_start + 148
3   libsystem_pthread.dylib       	0x00007fff6760083b thread_start + 15

Thread 3:
0   libsystem_pthread.dylib       	0x00007fff67600818 start_wqthread + 0

```

**Comment by koekeishiya** [comment]
Fixed on master.

**Comment by koekeishiya** [comment]
Fixed in 2.3.0
